src/sentry/templates/sentry/error-page-embed.js

```diff
@@ -13,9 +13,9 @@
     };
    */
 
-  var strings = /*{{ strings }}*/ '';
-  var template = /*{{ template }}*/ '';
-  var endpoint = /*{{ endpoint }}*/ '';
+  var strings = /* {{ strings }} */ '';
```

Comment:
spurious change?

Reply:
Must have been the autoformatter, but don't know what triggered it. Reverted this change just to be safe.